feature_flipper/templatetags/feature_flipper.py

```diff
@@ -57,4 +57,4 @@ def _get_value(self, key, context):
             return key[1:-1]
         if key in string.digits:
             return int(key)
-        return context[key]
+        return context.get(key, False)
```

Comment:
This is depending on the fact that `False` is `0` in Python 2 (which doesn't hold for Python 3). You'd be best changing this round a bit so there's a concept of there being no value at all here, then passing that information to the `render` method somehow.

If `_get_value` hasn't got a valid key then it shouldn't call `show_feature` at all.